models/AppState.py

```diff
@@ -1,5 +1,77 @@
 """Application state class"""
 
+import sys
+from numpy import array as np_array, min as np_min, ptp as np_ptp
+from models.PyCryptoBot import PyCryptoBot
+from models.TradingAccount import TradingAccount
+from models.exchange.binance import AuthAPI as BAuthAPI
+from models.exchange.coinbase_pro import AuthAPI as CAuthAPI
+
+def binanceMinimumOrderBase(app:PyCryptoBot, account:TradingAccount):
+    app = PyCryptoBot(exchange='binance')
+    api = BAuthAPI(app.getAPIKey(), app.getAPISecret(), app.getAPIURL())
+    info = api.client.get_symbol_info(symbol='BTCGBP')
+
+    base_min = 0
+    if 'filters' in info:
+        for filter_type in info['filters']:
+            if 'LOT_SIZE' == filter_type['filterType']:
+                base_min = float(filter_type['minQty'])
+
+    base = float(account.getBalance(app.getBaseCurrency()))
+
+    if base < base_min:
+        sys.tracebacklimit = 0
+        raise Exception(f'Insufficient Base Funds! (Actual: {base}, Minimum: {base_min})')    
+
+def binanceMinimumOrderQuote(app:PyCryptoBot, account:TradingAccount):
+    app = PyCryptoBot(exchange='binance')
+    api = BAuthAPI(app.getAPIKey(), app.getAPISecret(), app.getAPIURL())
+    info = api.client.get_symbol_info(symbol='BTCGBP')
```

Comment:
Hardcoded symbol.

Reply:
Same as previous but thanks for spotting it.